server/routerlicious/packages/routerlicious-base/src/nexus/runnerFactory.ts

```diff
@@ -210,10 +214,26 @@ export class NexusResourcesFactory implements core.IResourcesFactory<NexusResour
 			expireAfterSeconds: redisConfig2.keyExpireAfterSeconds as number | undefined,
 		};
 
-		const redisClient = new Redis.default(redisOptions2);
+		const redisClient: Redis.default | Redis.Cluster = redisConfig2.enableClustering
+			? new Redis.Cluster([{ port: redisConfig2.port, host: redisConfig2.host }], {
+					redisOptions: redisOptions2,
+					slotsRefreshTimeout: 50000,
```

Comment:
Should this be configurable?

Reply:
That's a good point, I had to modify this number a few times, I've moved it to the redisConfig in new commit